internal/transformer/xlinecluster_resource.go

```diff
@@ -83,8 +83,20 @@ func MakeStatefulSet(cr *xapi.XlineCluster, scheme *runtime.Scheme) *appv1.State
 		Ports: []corev1.ContainerPort{
 			{Name: "xline-port", ContainerPort: 2379},
 		},
+		Command: []string{
```

Comment:
We should use the args defined in XlineCluster to cover the default bootup command.

Reply:
done